.github/workflows/tests.yml

```diff
@@ -0,0 +1,60 @@
+name: tests
+
+on:
+  push: null
+  pull_request: null
+
+jobs:
+  tests:
+    name: tests
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: cancel previous runs
+        uses: styfle/cancel-workflow-action@0.6.0
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v2
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: 3.8
+          channels: conda-forge,defaults
+          channel-priority: strict
+          show-channel-urls: true
+
+      - name: configure conda and install code
+        shell: bash -l {0}
+        run: |
+          conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
+          conda update --quiet conda
+          conda config --add channels conda-forge --force
+          conda config --set channel_priority strict
+          conda config --set safety_checks disabled
+          conda create --name TEST python=$PY --file requirements.txt --file requirements-dev.txt
+          source activate TEST
+          conda info --all
+          python setup.py sdist && version=$(python setup.py --version) && pushd dist && pip install --no-deps --force-reinstall depfinder-${version}.tar.gz && popd
```

Comment:
Oh boy! I'm ashamed of this line now in light of so many better tools to test the tarball. PR to fix it as soon as you merge this.

Reply:
I'm happy to take pushes to this PR if you'd prefer.

Reply:
Better later b/c it will change the dependencies and we avoid multiple contexts with this PR. But if you want it is mostly tarball tests like:

```yaml
    - name: Tarball tests
      shell: bash -l {0}
      run: |
        pip wheel . -w dist --no-deps
        check-manifest --verbose
        twine check dist/*
```